README.md

```diff
@@ -1,13 +1,44 @@
 # Sample Revel Applications
 
-Each directory contains a revel application.
+Each sub directory contains a standalone [Revel](http://revel.github.io/manual)  application, eg
```

Comment:
Actually, why do we need "sub" at all? In the context of this repo, "each directory" is accurate.